### ISSUE TITLE
s_humanLSF - error related to wave, I think

### ISSUE THREAD

**Issue by wandell** [isetbio/isetbio#19]
This is a wavelength interpolation error ... one of you probably knows about this already, and it may even be in the Issues somewhere.  I added it here because I couldn't find Issue immediately.

Also ... a number of the s_<> scripts are not working now.  

Error in interp1 (line 191)
        F = griddedInterpolant(X,V,method);

Error in oiSet (line 253)
            p = interp1(oiGet(oi, 'wave')', p', val, 'linear*', 0);

Error in opticsSICompute (line 20)
oi = oiSet(oi, 'wave', sceneGet(scene, 'wave'));

Error in oiCompute (line 75)
        oi = opticsSICompute(scene,oi);

Error in s_humanLSF (line 35)
oi = oiCompute(scene550,oi);

**Comment by hjiang36** [comment]
Yes, this is caused by wavelength interpolation and I fixed the same problem in t_humanLineSpread. 

When we do oiCompute, we change the oi to have the same wavelength samples as the scene. In this script, we use narrow band scene (which only samples at 1 wavelength). After computation, the oi structure get interpolated to have only 1 wavelength samples. In the next run, when we input another narrow band scene, oiCompute try to interpolate to the new wavelength samples. However, the current oi structure only contains one wavelength and this makes the interpolation troublesome.

When I fix t_humanLineSpread, I changed the code to avoid overriding oi structure after oiCompute. That is using oi450S = oiCompute(scene, oi); instead of oi = oiComptue(scene, oi);

I'll think about how to fix the wavelength interpolation routine to avoid this problem.
